### PR TITLE
Updated declarations and If statement cleanliness

### PR DESCRIPTION
**Pull request by Sir-Kam**
changed -includes( . . . )- to -indexOf(...)-

### CODE REVIEW

File: src/Info.ts [hunk 1]
```diff
@@ -54,11 +54,11 @@ export default class Info {
         // if using .syntax we can only read notes
         let commandIsFetch = false;
```
Comment (stelar7) [inline src/Info.ts]: this can be removed and it can be made const where you assign it

File: src/Info.ts [hunk 2]
```diff
@@ -54,11 +54,11 @@ export default class Info {
...
 
-        // Needs to start with '/' or '!' or in separate cases '.'
+        // Needs to start with '/' / '!' or '.'
+        const contentPrefix = message.cleanContent[0];
```
Comment (stelar7) [inline src/Info.ts]: split[0][0]?

Comment (Sir-Kam) [reply src/Info.ts]: This gets the first char in the string. So the use of that was unnecessary. But either could be used, just in my opinion, i thought it looked messy; Though, I am re-referencing the cleanContent, when the same data is in 'split'.

**Comment by Querijn** [comment]
Thanks for your commit. Unfortunately I've taken a good hard look at Info and I see that things could be done far better. With the changes I made to include '.' as a prefix for fetch I should've split up this prefix into its own handler rather than in the same function as the !note handler. 

I believe that's the way forward for the note functionality. Next to that, it could use some other changes, making it dependent on the commands a lot for cleanliness.

**Comment by Sir-Kam** [comment]
Agreed.

**Pull request closed without merging**